src/views/VideoMania/index.jsx

```
@@ -19,27 +15,31 @@ import videoIcon from "../../assets/videoIcon.svg";
 import { nameElipse } from "../../utils/common/nameElipse";
 import Modal from "../../components/modal/Modal";
 import { AddVideoMania } from "../../services/videomania";
+import Player from "../../components/player";
 
 const VideoMania = () => {
   const [searchQuery, setSearchQuery] = useState("");
   const [currentPage, setCurrentPage] = useState(1);
```

ESLint

[error] 'currentPage' is assigned a value but never used.

(no-unused-vars)